Internal Server Error: /
Traceback (most recent call last):
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: indexa.html
"GET / HTTP/1.1" 500 79492
Internal Server Error: /
Traceback (most recent call last):
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader.py", line 42, in select_template
    return engine.get_template(template_name)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 514, in parse
    self.extend_nodelist(nodelist, compiled_result, token)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 531, in extend_nodelist
    raise self.error(
django.template.exceptions.TemplateSyntaxError: <ExtendsNode: extends "base.html"> must be the first tag in the template.
"GET / HTTP/1.1" 500 127003
Internal Server Error: /
Traceback (most recent call last):
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'endtrans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 132, in render
    compiled_parent = self.get_parent(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 129, in get_parent
    return self.find_template(parent, context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 107, in find_template
    template, origin = context.template.engine.find_template(
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 568, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 14: 'endtrans'. Did you forget to register or load this tag?
"GET / HTTP/1.1" 500 173207
Internal Server Error: /
Traceback (most recent call last):
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'endtrans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 132, in render
    compiled_parent = self.get_parent(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 129, in get_parent
    return self.find_template(parent, context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 107, in find_template
    template, origin = context.template.engine.find_template(
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 568, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 14: 'endtrans'. Did you forget to register or load this tag?
"GET / HTTP/1.1" 500 173207
Internal Server Error: /users/
Traceback (most recent call last):
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'endtrans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/vladislav94/python-project-52/task_manager/views.py", line 27, in users
    return render(request, 'users.html', context={'output': output})
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 232, in do_block
    nodelist = parser.parse(("endblock",))
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 558, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 5: 'endtrans', expected 'endblock'. Did you forget to register or load this tag?
"GET /users/ HTTP/1.1" 500 160591
Internal Server Error: /
Traceback (most recent call last):
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/defaulttags.py", line 1026, in find_library
    return parser.libraries[name]
KeyError: 'bootstrap4'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 132, in render
    compiled_parent = self.get_parent(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 129, in get_parent
    return self.find_template(parent, context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 107, in find_template
    template, origin = context.template.engine.find_template(
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/defaulttags.py", line 1088, in load
    lib = find_library(parser, name)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/defaulttags.py", line 1028, in find_library
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'bootstrap4' is not a registered tag library. Must be one of:
admin_list
admin_modify
admin_urls
cache
i18n
l10n
log
static
tz
"GET / HTTP/1.1" 500 179725
Internal Server Error: /
Traceback (most recent call last):
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/defaulttags.py", line 1026, in find_library
    return parser.libraries[name]
KeyError: 'bootstrap4'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 132, in render
    compiled_parent = self.get_parent(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 129, in get_parent
    return self.find_template(parent, context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 107, in find_template
    template, origin = context.template.engine.find_template(
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/defaulttags.py", line 1088, in load
    lib = find_library(parser, name)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/defaulttags.py", line 1028, in find_library
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'bootstrap4' is not a registered tag library. Must be one of:
admin_list
admin_modify
admin_urls
cache
i18n
l10n
log
static
tz
"GET / HTTP/1.1" 500 179725
Internal Server Error: /
Traceback (most recent call last):
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/defaulttags.py", line 1026, in find_library
    return parser.libraries[name]
KeyError: 'bootstrap4'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 132, in render
    compiled_parent = self.get_parent(context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 129, in get_parent
    return self.find_template(parent, context)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 107, in find_template
    template, origin = context.template.engine.find_template(
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/defaulttags.py", line 1088, in load
    lib = find_library(parser, name)
  File "/home/vladislav94/python-project-52/.venv/lib/python3.10/site-packages/django/template/defaulttags.py", line 1028, in find_library
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'bootstrap4' is not a registered tag library. Must be one of:
admin_list
admin_modify
admin_urls
cache
i18n
l10n
log
static
tz
"GET / HTTP/1.1" 500 176949
